README.md

```diff
@@ -97,6 +97,14 @@ Check [capabilities](https://www.browserstack.com/automate/capabilities) in Brow
 
 #### Other configurations
 
+##### Screen resolution
```

Comment:
I think if we add this new feat. (specifying width, height) in remote we should also add it for local as it is also a valid feature for local.

Reply:
I don't think it is a valid feature for local as you can't really change the resolution of the screen you have.

Reply:
It is confusing, but this helped me - screen is the device & window is the browser. I can adapt the naming to be a bit clearer